File: ruby/old/fizzbuzz.rb
1.upto(1000) do |n|
    tmp = 0
    n.to_s.each_char{|n|tmp += n.to_i}
    str = "#{'fizz ' if /(0|5)/ =~ n.to_s[-1]}#{'buzz' if tmp == 3}"
    puts str if '' != str
    puts n.to_s if '' == str
 end

File: c/fizzbuzz.c
#include <stdio.h>

int main() {
  int i;
  for(i=0;i <= 100;i++) {
    if((i % 3) == 0) {
      printf("Fizz");
    }
    if((i % 5) == 0) {
      printf("Buzz");
    }
    if((i % 5) != 0 && (i % 3) != 0) {
      printf("%d",i);
    }
    printf("\n");
  }
  return 0;
}
